recipe/bld.bat

```diff
@@ -6,41 +6,39 @@ del cmake_modules\SetupCxxFlags.cmake
 del cmake_modules\CompilerInfo.cmake
 del cmake_modules\FindNumPy.cmake
 del cmake_modules\FindPythonLibsNew.cmake
+del cmake_modules\ArrowDatasetConfig.cmake
 copy /Y "%SRC_DIR%\cpp\cmake_modules\BuildUtils.cmake" cmake_modules\
 copy /Y "%SRC_DIR%\cpp\cmake_modules\SetupCxxFlags.cmake" cmake_modules\
 copy /Y "%SRC_DIR%\cpp\cmake_modules\CompilerInfo.cmake" cmake_modules\
 copy /Y "%SRC_DIR%\cpp\cmake_modules\FindNumPy.cmake" cmake_modules\
 copy /Y "%SRC_DIR%\cpp\cmake_modules\FindPythonLibsNew.cmake" cmake_modules\
+copy /Y "%SRC_DIR%\cpp\cmake_modules\ArrowDatasetConfig.cmake" cmake_modules\
 
 SET ARROW_HOME=%LIBRARY_PREFIX%
 SET SETUPTOOLS_SCM_PRETEND_VERSION=%PKG_VERSION%
 SET PYARROW_BUILD_TYPE=release
-SET PYARROW_WITH_COMPUTE=1
-SET PYARROW_WITH_CSV=1
+SET PYARROW_WITH_ACERO=1
 SET PYARROW_WITH_S3=1
+SET PYARROW_WITH_SUBSTRAIT=1
 SET PYARROW_WITH_HDFS=1
 SET PYARROW_WITH_DATASET=1
-SET PYARROW_WITH_FILESYSTEM=1
 SET PYARROW_WITH_FLIGHT=1
 SET PYARROW_WITH_GANDIVA=0
-SET PYARROW_WITH_JSON=1
 SET PYARROW_WITH_ORC=1
 SET PYARROW_WITH_PARQUET=1
 SET PYARROW_WITH_PARQUET_ENCRYPTION=1
-@REM Plasma doesn't actually get built for Windows in arrow
-SET PYARROW_WITH_PLASMA=0
 SET PYARROW_CMAKE_GENERATOR=Ninja
 SET PYARROW_WITH_CUDA=0
 @rem manually specify the path to the cmake config files
 SET Arrow_DIR=%ARROW_HOME%\cmake\Arrow
 SET ArrowFlight_DIR=%ARROW_HOME%\cmake\ArrowFlight
 SET ArrowDataset_DIR=%ARROW_HOME%\cmake\ArrowDataset
+SET ArrowAcero_DIR=%ARROW_HOME%\cmake\ArrowAcero
 SET Parquet_DIR=%ARROW_HOME%\cmake\Parquet
+SET ArrowSubstrait_DIR=%ARROW_HOME%\cmake\ArrowSubstrait
+
+"%PYTHON%" -m pip install . -vv --no-deps --no-build-isolation --global-option="build_ext" --global-option="--single-version-externally-managed" --global-option="--record=record.txt"
```

Comment:
```suggestion
"%PYTHON%" -m pip install . -vv --no-deps --no-build-isolation
```
I don't think `--single-version-externally-managed` and `--record=record.txt` are required.